config/common-config.yaml

```diff
@@ -595,6 +595,10 @@ ingressNginx:
       ## ref: https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/openstack-cloud-controller-manager/expose-applications-using-loadbalancer-type-service.md#restrict-access-for-loadbalancer-service
       loadBalancerSourceRanges: []
 
+      # Set this to use an existing floating ip
+      # If you want to change this then you need to recreate the service
+      loadBalancerIP: ""
```

Comment:
Add this to the schema.

Reply:
Done just now
Please check if it is ok
Thank you!